glottolog3/models.py

```diff
@@ -541,6 +541,39 @@ class LanguoidSchema(Schema):
     def make_languoid(self, data):
         return Languoid(**data)
 
+def validate_id_type(type):
+    if (type != IdentifierType.iso.value and \
+        type != IdentifierType.glottolog.value and \
+        type != IdentifierType.wals.value and \
+        type != IdentifierType.ethnologue.value and \
+        type != 'name'):
+        raise ValidationError('Invalid identifier type: {}'.format(type))
+   
+class IdentifierSchema(Schema):
+    id = fields.Str(default = '')
+    name = fields.Str(required=True)
+    type = fields.Str(required=True, validate=validate_id_type)
+    description = fields.Str(required=True)
+    lang = fields.Str(required=True) 
```

Comment:
so description and lang are still required? Couldn't we default description to an empty string, and default lang to `en` (or however glottolog does it)?